src/jvm/test-trace-debugger/org/jetbrains/kotlinx/lincheck_test/util/Utils.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.kotlinx.lincheck_test.util
+
+import org.junit.Ignore
+
+typealias IgnoreInTraceDebuggerMode = Ignore
```

Comment:
Please explain in the code the magic here

Reply:
Done